src/ui/components/LedgerViewer.js

```diff
@@ -222,13 +222,22 @@ const LedgerEventRow = React.memo(
         case "ADD_ALIAS":
           try {
             const account = ledger.account(action.identityId);
-            // description has format: channel/[@handle](url). url has little value in the UI, so cut it
-            const aliasHandle = action.alias.description.replace(/\(.*\)$/, "");
+            let alias = action.alias.description;
+            // description has format: channel/[@handle](url). Parse out alias, account, url
+            const descriptionParts = alias.match(
+              /^([^/]*)\/\[(@[^\]]*)\]\(([^)]*)\)$/
```

Comment:
What a regex we need for this 😂 

(In my head I thought of just stripping `[` and `]` from the output since we don't allow them as characters in identity, but this is obviously much more robust)